src/AirTable_CompanyParcer/parcer.py

```diff
@@ -31,7 +32,7 @@
 URL_prev = "https://airtable.com/appzLUHyhTU5xpkdZ/shrclnXK0pfoGjtih/tblQ0U46nsYopm2CR"
 columns = ["id", "Company", "Location HQ", "# Laid Off", "Date", "%", "Industry",
            "Source", "List of emploees laid", "Stage", "$ Raised mm", "Country", "Date added"]
-dir = "src/"
```

Comment:
you shouldn't touch this line. and also you have to add the same line in  the .ipynb file. you have to find absolute path of source files because if use venv it sees path differently from just python.